install/dataloggers.csv

```diff
@@ -1,17 +1,20 @@
 Make,Model,Serial,Place,Role,Start Date,End Date
 Campbell Scientific,CR6,03066,Lower Te Maari Fumarole 15,,2018-08-10T00:00:00Z,9999-01-01T00:00:00Z
+Campbell Scientific,CR6,04108,Ruapehu Crater Lake Outlet,,2022-01-17T22:40:00Z,9999-01-01T00:00:00Z
 Campbell Scientific,CR6,04109,White Island Fumarole 13,,2018-09-13T01:30:00Z,9999-01-01T00:00:00Z
 Campbell Scientific,CR6,04110,Upper Te Maari Fumarole 6,,2018-08-28T00:00:00Z,9999-01-01T00:00:00Z
 Campbell Scientific,CR6,04111,Fox Glacier Alpine Gardens,,2019-02-06T00:00:00Z,2021-05-05T20:50:01Z
 Campbell Scientific,CR6,04114,Fox Glacier Alpine Gardens,,2021-05-05T22:09:59Z,9999-01-01T00:00:00Z
-Campbell Scientific,CR6,04115,Ruapehu Crater Lake Outlet,,2018-05-03T00:00:00Z,9999-01-01T00:00:00Z
+Campbell Scientific,CR6,04115,Ruapehu Crater Lake Outlet,,2018-05-03T00:00:00Z,2022-01-17T22:39:59Z
+Campbell Scientific,CR6,04115,Tongariro Lower Emerald Spring,,2022-03-30T21:10:00Z,9999-01-01T00:00:00Z
 Campbell Scientific,CR6,04116,Waimangu Inferno Crater,,2017-11-07T00:00:00Z,9999-01-01T00:00:00Z
-Campbell Scientific,CR6,04117,Upper Tama Lake,,2019-03-12T03:30:00Z,9999-01-01T00:00:00Z
+Campbell Scientific,CR6,04117,Upper Tama Lake,,2019-03-12T03:30:00Z,2022-03-30T19:39:59Z
```

Comment:
Hi @rumachan , thanks a lot for sharing the data, really helpful!
should this closing time be some time after the last data point received from the old logger?
`"2022-03-30 20:30:00",10364,"TOUTL",1.53,1.53,1.533,12.87,12.83,12.87,12.7,12.66,12.7`

so with `2022-03-30T20:40:00Z` as starting date of new data logger, end date here to be
`2022-03-30T20:39:59Z` (or playing with minutes/seconds between 20:30 and 20:40 in case we have data gaps in between?)

Reply:
That's a good question. The times are inclusive aren't they? So as long as there is no overlap and no data points fall outside the installed periods we are okay? I don't have the times of exactly when a logger was disconnected as it's not recorded. Just setting date-time to fit the data.

Samples are collected at 10 minute intervals so there can't be a data gap between 20:30 and 20:40, just data values at 20:30 and 20:40.

Reply:
yes, times are inclusive. The "approach" I was using for gnss was to have a start/stop timestamp during the data gap, to ensure that we were covering all available data with associated metadata. 
(and btw: good on the field technicians that can swap a datalogger within a 10 minutes interval...that's really impressive!)

and to summarise, the main change I was suggesting here was to: 
```diff
-Campbell Scientific,CR6,04117,Upper Tama Lake,,2019-03-12T03:30:00Z,2022-03-30T19:39:59Z
+Campbell Scientific,CR6,04117,Upper Tama Lake,,2019-03-12T03:30:00Z,2022-03-30T20:39:59Z
```

if you agree that the 04117 logger was removed sometime between 20:30 and 20:40

Reply:
So I made a mistake with the end time? Hour should be 20, but I had 19? Now I get it

Reply:
he he yes, sorry my bad english! yes, that's what I meant (but I was also asking question about the data and swap procedure at the same time :stuck_out_tongue: